docs/glossary.md

```diff
@@ -0,0 +1,138 @@
+
+See also:
+
+- [General Instruction of the Liturgy of the Hours](#general-instruction-of-the-liturgy-of-the-hours).
+
+## Precedence
+
+The principle of precedence is used to determine which liturgical celebration should be observed when two or more celebrations fall on the same day. The principle of precedence is based on the liturgical rank of the celebrations.
+
+See [GNLYC §59](https://www.liturgyoffice.org.uk/Resources/GIRM/Documents/GNLY.pdf#page=14).
+
+## Proper of Saints
+
+The Proper of Saints (or Sanctoral) consists of the fixed feasts, celebrated on the very same date each year (no matter what the day of the week),including Christmas and all the saints’ days.
+
+## Proper of Time
+
+The Proper of Time (or Temporal) consists of the movable feasts, most of them keyed to Easter (which falls on a different Sunday every year), including Ascension, Pentecost, and so on.
+
+> [!NOTE]
+> We avoid using the term Temporal to prevent potential confusion with the `Temporal` object in ECMAScript, which is still in a [proposal stage](https://tc39.es/proposal-temporal/docs/).
+
+## Psalter Week Cycle
+
+The four-week cycle of the psalter is coordinated with the liturgical year in such a way that on the First Sunday of Advent, the First Sunday in Ordinary Time, the First Sunday of Lent, and Easter Sunday the cycle is always begun again with Week 1 (others being omitted when necessary).
+
+See [GILH §133](https://www.liturgyoffice.org.uk/Resources/Rites/GILH.pdf#page=26).
+
+## Rank
+
+The rank of a liturgical celebration is a measure of its importance in the liturgical calendar. The rank of a celebration determines how it should be observed when it falls on the same day as another celebration.
+
+There are five categories of ranks: solemnities, Sunday, feasts, memorials, weekday. A memorial can be obligatory or optional.
+
+See [GNLYC §3-16](https://www.liturgyoffice.org.uk/Resources/GIRM/Documents/GNLY.pdf#page=5).
+
+## Sanctoral
+
+See [Proper of Saints](#proper-of-saints).
+
+## Season
+
+Liturgical seasons are the periods of time in the liturgical year that are marked by a particular liturgical color and have a particular focus. The liturgical seasons are Advent, Christmastide, Lent, Easter Time, and Ordinary Time. The Ordinary Time is divided into two parts: the first part begins on the Monday after the Baptism of the Lord and ends on the day before Ash Wednesday; the second part begins on the Monday after Pentecost and ends on the Saturday before the First Sunday of Advent.
+
+## Sunday Cycle
+
+A three-year cycle for Sunday Mass readings (and some solemnities), designated by A, B, or C. Each cycle begins on the First Sunday of Advent of the previous civil year and ends on Saturday after the Christ the King Solemnity. The cycles follow each other in alphabetical order. C year is always divisible by 3, A has remainder of 1, and B remainder of 2.
```

Comment:
it's talking about the numerical property "year" of C, because `Year C's numerical value` would be a bit more wordy than is worth it to me. also, it doesn't really matter to me in this case

Reply:
If `year` refers to a property name, IMO it should be styled as a code (using backticks), however, in this case it specifically talks about Year C (or _every Year C year_ :wink:).

Now, I think that sentence should be rephrased. How about this?

```suggestion
A three-year cycle for Sunday Mass readings (and some solemnities), designated by A, B, or C. Each cycle begins on the First Sunday of Advent of the previous civil year and ends on Saturday after the Christ the King Solemnity. The cycles follow each other in alphabetical order. Year C is always the year divisible by 3, while Year A leaves a remainder of 1, and Year B a remainder of 2.
```

Reply:
i think we can discuss these semantics later and all this falls into "further iterations" and doesn't need to be exact this second.

Reply:
Okay.